USERS.md

```diff
@@ -203,6 +203,11 @@ Users (Alphabetically)
       D: Radio France is using Cilium in their production clusters (self-hosted k8s with kops on AWS)
       U: Mainly Service load-balancing
       Q: @francoisj
+      
+    * N: Rancher Labs, now part of SUSE
+      D: Rancher Labs certified Kubernetes distributions, RKE and RKE Government, both use Cilium as one of their supported CNIs.
```

Comment:
Please update to `Rancher Labs certified Kubernetes distribution RKE2 can be deployed with Cilium.`

Reply:
Will do :)